docs/en/connector-v2/sink/Hudi.md

```diff
@@ -115,9 +115,9 @@ Note: When this configuration corresponds to a single table, you can flatten the
 
 `max_commits_to_keep` The max commits to keep of hudi table.
 
-### auto_commit [boolean]
```

Comment:
why delete `auto_commit`?

Reply:
> why delete `auto_commit`?

1.HoodieJavaWriteClient is auto commit by default when not configure this.
2.Due to the nature of hudi timelines, data needs to be submitted immediately after the data is writed or deleted, If not next flush will not aware data state of the previous batch. This may result in duplicate data or loss of deleted data.